documentation/docs/developers/connectors/task3.md

```diff
@@ -505,30 +554,77 @@ Use `OrderType.LIMIT_MAKER` to specify you want a trading fee for the maker orde
 | `quote_currency` | `str`                                                                                                      | Quote currency of the order.                                                              |
 | `price`          | `Decimal`                                                                                                  | Price in which the order will be placed in `Decimal`                                      |
 | `amount`         | `Decimal`                                                                                                  | Amount in which the order will be placed in `Decimal`                                     |
-| `order_type`     | [`OrderType`](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/core/event/events.py#L61-L69) | Specifies the order type of the order(i.e. `OrderType.LIMIT` and `OrderType.LIMIT_MAKER`) |
-| `trade_type`     | [`TradeType`](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/core/event/events.py#L61-L69) | Specifies the trade type of the order(i.e. `TradeType.BUY` and `TradeType.SELL`)          |
+| `order_type`     | [`OrderType`](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/core/event/events.py#L72-L78) | Specifies the order type of the order(i.e. `OrderType.LIMIT` and `OrderType.LIMIT_MAKER`)  |
+| `trade_type`     | [`TradeType`](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/core/event/events.py#L66-L69) | Specifies the trade type of the order(i.e. `TradeType.BUY` and `TradeType.SELL`)           |
 
 **Expected Output(s):**
 
 | Output(s) | Type                                                                                                        | Description                       |
 | --------- | ----------------------------------------------------------------------------------------------------------- | --------------------------------- |
-| `fee`     | [`TradeFee`](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/core/event/events.py#L257-L287) | Estimated trade fee of the order. |
+| `fee`     | [`TradeFee`](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/core/event/events.py#L272-L302) | Estimated trade fee of the order. |
+
+### (7) Additional Derivative Functions
 
-### (6) Class Properties
+Below are the additional methods that `Derivative` class must implement.
 
-Below are the property functions of the `Exchange` class.
+### `_update_account_positions()`
+
+Ensures the `_account_positions` dictionary is in sync with the information in the exchange.
```

Comment:
Would be good to mention that it should be one of the functions being called in the `status_polling_loop()` function.